tests/manage/nfs_feature/test_nfs_feature_enable_for_ODF_clusters.py

```diff
@@ -267,6 +319,20 @@ def test_outcluster_nfs_export(
         13:- Deletion of Pods and PVCs
 
         """
+        if not self.nfs_client_ip:
+            pytest.skip(
+                "Skipped the test as a valid nfs client ip is required, "
+                " for nfs outcluster export validation. "
+            )
+
+        # ssh to test-nfs-vm
+        log.info("Login to test vm")
+        self.con = Connection(
```

Comment:
You are invoking this connection 4 times in this module, why not to have it in init of the class and re-use same connection in all places?

If the connection is not stable or there is some timeout, you can have slef.con define as property, and this property will check if connection exist, if so it will retun it from self._con e.g. and if it doesn't exist or it's expired or whatever it will create new connection and return new one.

Reply:
I have kept it like tha because I have incluster and out of cluster tests in the same test class and created the connection details for the out cluster tests. And  have added a check to skip the test in case valid ip address is not available for the out cluster mount validation tests.